keystore/keystore.go

```diff
@@ -439,17 +439,29 @@ type ServerKeyStore interface {
 	Reset()
 }
 
+// KeyMarker set key market for KeyDescription
+type KeyMarker string
```

Comment:
lets name it as "state", which may be current or rotated. Because "marker" is not obvious meaning in that context